defaults/main.yml

```diff
@@ -2,3 +2,6 @@
 ---
 cron_apt_mailto: root
 cron_apt_mailon: upgrade
+
+cron_apt_install:
+  - liblockfile-bin
```

Comment:
We can also make it part of the dependencies, if you think it is one. Or do you think there's a use case not to install it?

Reply:
Yeah, that's probably better. I will update this PR.